src/koans/14_recursion.clj

```diff
@@ -3,21 +3,30 @@
 
 (defn is-even? [n]
   (if (= n 0)
-    __
-    (___ (is-even? (dec n)))))
+    true
+    (not (is-even? (dec n)))))
 
 (defn is-even-bigint? [n]
   (loop [n   n
          acc true]
     (if (= n 0)
-      __
+      false
       (recur (dec n) (not acc)))))
 
 (defn recursive-reverse [coll]
-  __)
+  (loop [coll coll
+         acc '()]
+  (if (= coll [])
+    acc
+    (recur (rest coll) (conj acc (first coll) )))))
 
 (defn factorial [n]
-  __)
+  (loop [number n
+         acc 1]
+    (if (= number 0) 
```

Comment:
It's enough to check if number is equal 1 because you don't need to multiply the whole thing by one again, result will be the same.

Reply:
Yea, nice call.